README-Rus-ru.md

```
@@ -0,0 +1,659 @@
+<div align="center">
+  <img height="128" src="assets/logo.png" alt="Lenovo Legion Toolkit Logo">
+  
+  # Lenovo Legion Toolkit
```

markdownlint-cli2 (0.22.1)

[warning] Headings must start at the beginning of the line

(MD023, heading-start-left)